Tasks/DeployAzureResourceGroup/task.json

```diff
@@ -325,13 +372,22 @@
         "FailedToFetchDetailsOfLB": "Failed to fetch the details of the load Balancer: %s",
         "FailedToUpdateLBInboundNatRules": "Failed to update the inbound NAT rules of the Load Balancer %s to remove irrelevant rules",
         "InvalidResponseLongRunningOperation": "Invalid response received for fetching status of a long running operation.",
-        
         "AddingInboundNatRule": "Adding Inbound Nat Rule for the Virtual Machine %s to the Load Balancer %s",
         "InboundNatRuleAdditionFailed": "Addition of Inbound Nat Rule to the Load Balancer %s failed. Error: %s ",
         "InboundNatRulesToNICFailed": "Addition of rule Id to the loadBalancerInboundNatRules of nic %s failed with the error: %s",
         "AddedTargetInboundNatRuleLB": "Added the target virtual machine for the inbound NAT rules of load balancer %s",
         "FailedToFetchInstanceViewVM": "Failed to fetch the instance view of the virtual machine. Error : %s",
         "CreationOfExtensionFailed": "Failed to add the extension. Error: %s",
-        "AddedExtension" :"Successfully added the CustomScriptExtension for the virtual Machine %s"
+        "AddedExtension" :"Successfully added the CustomScriptExtension for the virtual Machine %s",
+        "OperationSucceeded": "%s of %s on vm %s succeeded",
+        "OperationFailed": "%s of %s on vm %s failed",
+        "MGAgentOperationOnAllVMsFailed": "Machine group agent %s did not succeed on all VMs. %s",
+        "MGAgentOperationOnAllVMsSucceeded": "Machine group agent %s succeeded on all VMs",
+        "PublicSettings": "Public settings are:\n VSTSAccountName: %s\nTeamProject: %s\nMachineGroup: %s\nTags: %s\n",
```

Comment:
If it is debug log, you can directly put in the file since that doesn't require localization.